src/gui/mzroll/pollyelmaveninterface.cpp

```diff
@@ -277,9 +281,9 @@ void PollyElmavenInterfaceDialog::startupDataLoad()
     }    
```

Comment:
@shubhra-agrawal 

First of all, this function should be startupDataLoad(QVariantMap projectNamesId)
second, 
If projectNamesId.isEmpty(), then we have no internet connection issue and the same should be communicated to the user instead of trying to get user projects again.

This will resolve comments made by @rish9511 and @saifulbkhan regarding EPI returning "ok" even if there is no internet connection.

By the way, this was happening because -

- Once anyone logs into Elmaven, it will store the token information for persistent login.

- Now, while authenticating it will check for token expiry. If someone tries to use EPI before one hour (token expiry time), it will return "ok" because actually it has all the authentication information to connect to Polly.

Reply:
@Ky5595 The problem with it always sending 'ok' should be resolved by this. Making those changes.
The error reported by @rish9511 and @saifulbkhan happened further ahead on Polly. Will relay that to you on personal chat.